src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -54,6 +54,16 @@ public bool VisitInlineCommand(InlineCommandComment comment)
 
         public bool VisitParagraphCommand(ParagraphComment comment)
         {
+            if (comment.Content.Any(k => k.Kind == DocumentationCommentKind.InlineCommandComment))
+            {
+                foreach (var item in comment.Content.Where(c => c.Kind == DocumentationCommentKind.TextComment))
```

Comment:
I think this code checks all text comments rather than only those following a command. Shouldn't we only change the latter?